app/services/DR/getToken.js

```diff
@@ -30,3 +32,27 @@ export default async function getToken(needNewToken = false) {
     }
   }
 }
+
+export async function getTokenMepid(needNewToken = false) {
+  const savedToken = await GetStoreData('MEPID_API_TOKEN', true);
+
+  const mepidUrl = 'https://covidrdapi.msp.gob.do:8443';
+  const mepidUsername = 'covidrd';
+  const mepidPassword = 'msp@2020app';
```

Comment:
This credentials should go in the **.env** file and call it from baseUrls. Please delete it